extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -332,7 +332,8 @@ public function testGetContext(
 				'pageType' => 'all_ads',
 				'showAds' => true,
 				'delayBtf' => true,
-				'sourcePointRecovery' => false
+				'sourcePointRecovery' => false,
+				'sourcePointMMS' => false
 			],
 			'targeting' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "targeting" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)